### DISCUSSION TITLE
Use media-stream-library-js to play AXIS camera recordings

### DISCUSSION THREAD

**Discussion by AlessandroMartinelli** [AxisCommunications/discussions#173] [VAPIX] [answered]
Hi, 
I was able to use example [example-streams-web/camera/simple.html](https://github.com/AxisCommunications/media-stream-library-js/blob/main/example-streams-web/camera/simple.html) from [media-stream-library-js](https://github.com/AxisCommunications/media-stream-library-js) to play a recording stored on an AXIS Camera SD card directly on browser. 
Actually it's quite simple, in order to test this you just need to edit line 
https://github.com/AxisCommunications/media-stream-library-js/blob/07f9471495bdbf4e9a6c61c9e11a0d6c51794f05/example-streams-web/camera/simple-player.js#L44
by adding ```?recordingid=<recordingId>``` at the end of it.

This way the recording is played successfully in browser.

Now I'm trying to let the user navigate back and forth in the recording video stream sent from the camera. Using the video seek bar you can only seek to parts of the stream that have been buffered, so I thought I just need to send the command of playing the recording specifying as start time the timestamp I want the playback to start. From [AXIS VAPIX Documentation](https://www.axis.com/vapix-library/subjects/t10175981/section/t10004596/display?section=t10004596-t10004754) I understand I need to add an additional header like ```Range: ntp=<starttime>```.

However it's not clear to me how am I supposed to edit the above javascript file to add the Range header.

Could you please help me out?

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @AlessandroMartinelli! I think you will have a better chance of getting an answer if you write this question in the [media-stream-library-js](https://github.com/AxisCommunications/media-stream-library-js) repository.

**Reply by AlessandroMartinelli** [reply]
Hi @mattias-kindborg-at-work and thank you for the answer.

Actually I've written the [question](https://github.com/AxisCommunications/media-stream-library-js/issues/728) on media-stream-library-js repository even before writing here but, since I've received no answer in 4 days, and I really need to understand if this can be done or not, I though I would have had more chances of being answered here.

**Reply by mattias-kindborg-at-work** [reply]
I'm sorry, your best chance of getting an answer would be your original question.

**Reply by AlessandroMartinelli** [reply]
Ok, thanks.
